static/app/views/monitors/details.tsx

```diff
@@ -38,7 +44,16 @@ function MonitorDetails({params, location}: Props) {
     {query: {...location.query, environment}},
   ] as const;
 
-  const {data: monitor} = useApiQuery<Monitor>(queryKey, {staleTime: 0});
+  const {data: monitor} = useApiQuery<Monitor>(queryKey, {
+    staleTime: 0,
+    refetchInterval: data => {
+      if (!data) {
+        return false;
+      }
+      const [monitorData] = data;
+      return hasLastCheckIn(monitorData) ? false : DEFAULT_POLL_INTERVAL_MS;
```

Comment:
Let's throw a comment in here to indicate that this is just so it waits for the first checkin to come in

Reply:
boom did it 💣